packages/scss/src/components/inputs/textfield/_textfield.scss

```diff
@@ -1,54 +1,71 @@
 .textfield {
-	padding-top: _component("textfield.padding-top");
+	display: flex;
+	flex-direction: column;
 	position: relative;
 	width: _component("textfield.sizes.default");
 }
 
 .textfield-input {
-	background: transparent;
+	background: _component("textfield.background");
 	border: 0;
-	box-shadow: 0 1px 0 0 #CCCCCC;
-	color: _color("text.default");
+	border-radius: _component("textfield.border-radius");
+	box-shadow: 0 0 0 1px _component("textfield.border.color");
+	color: _component("textfield.input.color");
 	font-family: unquote(_theme("commons.font.family"));
 	font-size: _component("textfield.font-size");
-	font-weight: 400;
-	margin-bottom: _component("textfield.input.margin-bottom");
 	outline: 0;
-	padding: _component("textfield.input.padding-vertical") 0;
+	padding: _component("textfield.input.padding-vertical") _component("textfield.input.padding-horizontal");
 	position: relative;
 	transition: box-shadow 150ms ease, color 150ms ease;
 	width: 100%;
 
 	&::placeholder {
 		color: _color("text.placeholder");
-		opacity: 0;
 		transition: opacity 150ms ease;
 	}
 
 	&:focus {
-		&::placeholder {
-			opacity: 1;
-		}
-
-		~ .textfield-label {
-			font-size: _theme("sizes.small.font-size");
-			top: 0;
-		}
+		box-shadow: 0 0 0 1px _component("textfield.border.focus");
 	}
 }
 
+.textfield-suffix {
+	bottom: _component("textfield.input.padding-vertical");
```

Comment:
We should anchor it from the top and handle the `mod-noLabel` to avoir any issue with textfield-messages and other stuff that could extend the size of the `textfield`